presidio-analyzer/presidio_analyzer/predefined_recognizers/azure_ai_language.py

```diff
@@ -45,6 +47,7 @@ def __init__(
             supported_language=supported_language,
             name="Azure AI Language PII",
             version="5.2.0",
+            **kwargs
```

Comment:
Man I hate kwargs :)

Reply:
Me too! They are not here to pass things, just to conform with the parent class intrrface...